docs/learn-crosschain.md

```diff
@@ -26,7 +30,36 @@ messages.
 - Validators will check a proof that the new candidate for the next parachain block includes the
   processing of the expected ingress messages to that parachain.
 
-## Example
+XCMP queues must be initiated by first opening a channel between two parachains. The channel is
+identified by both the sender and recipeient parachains, meaning that it's a one-way channel. A pair
+of parachains can have at most two channels between them, one for sending messages to the other
+chain and another for receiving messages. The channel will require a deposit in DOT to be opened,
+which will get reciprocated when the channel is closed.
+
+## Horizontal Relay-routed Message Passing (HRMP)
+
+While XCMP is still being implemented, a stop-gap protocol (see definition below) known as HRMP
+exists in its place. HRMP has the same interface and functionality as XCMP but is much more
+demanding on resources since it stores all messages in the Relay Chain storage. When XCMP has been
+implemented, HRMP is planned to be deprecated and phased out in favor of it.
```

Comment:
```suggestion
implemented, HRMP is planned to be deprecated and phased out in favour of it.
```

Reply:
We use American spelling so `favor` is OK